lib/top/ocaml_topexpect.ml

```diff
@@ -0,0 +1,319 @@
+(*
```

Comment:
rename this file to something else, as it hasn't anything to do with ocaml_topexexpt anymore. Maybe `part.ml` with 2 exposed functions: `find` and `replace`?